Styling changes to the CC Libraries bar top

DivyaPrabhakar opened the pull request:
- Added an underline to "My Library" so that it signifies it is a datalist
Regarding #2108 

- Took away the bottom border for the title bar because it was looking too crowded with both the underline and the bottom border. Got designs from @davealonzo 

- Changed the height of the title to 5rem as per the design. Should we choose to add the bottom border again and take away the underline, the height of the top bar will still be the correct size. https://trello.com/c/jRlxiRui/491-design

shaoshing commented:
Thanks @DivyaPrabhakar . By removing the bottom border, It seems like unbalanced between the top border and list underline (is it just me?). We might have to also think about the design when creating / renaming a library. You can rebase master to import these new features into your branch. 

<img width="385" alt="screen shot 2015-08-21 at 9 23 54 am" src="https://cloud.githubusercontent.com/assets/118264/9413656/1a5fb168-47e8-11e5-92e9-72209321b00b.png">

<img width="331" alt="screen shot 2015-08-21 at 9 36 46 am" src="https://cloud.githubusercontent.com/assets/118264/9413666/26315afa-47e8-11e5-9843-e3b2cdad848e.png">

designedbyscience commented:
Can we just kill both of those hairlines? We are moving away from the hairlines in general. Maybe make the following 3 buttons have even space to that drop down (rather than align right?)

davealonzo commented:
@designedbyscience We can certainly can kill the hairline below the "LIBRARIES" title to match the Layers panel. But we should keep the blue one that invokes for create or rename library workflows

DivyaPrabhakar commented:
@davealonzo @designedbyscience @shaoshing  

So the consensus here is to:

1. Kill the top and bottom hairlines, and only have the underline that indicates a datalist

2. Add the underline to the bottom of My Library datalist 
![alt text](https://cl.ly/image/1m000W2R0u3Y?_ga=1.78146644.1165721305.1440120073 "hover text")

3. About the evenly spacing the icons part, they are evenly spaced already because the width of the data list is pretty long. You can see that when we add the underline -- the underline goes all the way to the Share icon. Should I shorten the datalist width and then spread out the icons?

volfied commented:
@DivyaPrabhakar in case you're waiting on this, this PR needs rebase.

DivyaPrabhakar commented:
@davealonzo @designedbyscience @shaoshing Once I get a confirmation and answer on the comment above about the final changes, I will make the changes, rebase and have these changes be merged.

iwehrman commented on src/style/sections/libraries/libraries-section.less:
```diff
@@ -90,17 +90,24 @@
             flex-grow: 1;
             border: 0;
-
+           border-bottom: @hairline solid fade(@color-subsection-rule, 60%);
```
You appear to be missing exactly one space here :space_invader:

iwehrman commented:
Should the datalist text be flush left here? 
![flush](https://s3.amazonaws.com/f.cl.ly/items/2L1j1r1x1t2M0i3g2T30/Screen%20Recording%202015-08-24%20at%2006.45%20PM.gif)

DivyaPrabhakar commented:
Talked to @davealonzo about how the UI should look regarding @iwehrman's question about datalist alignments. He approved the changes that I made which now should look like this: 
 
![alt text](https://www.dropbox.com/s/ff8xhno795354ce/Screenshot%202015-08-25%2011.52.38.png?raw=1)

- Removed hairlines
- Datalist flush left
- Extended width of datalist to reach icons
- Icons are flush right 
- Underlined the datalist

![alt text](https://www.dropbox.com/s/d1736tgar8prnwx/Screenshot%202015-08-25%2011.52.54.png?raw=1)

- Added padding to the bottom of the dropdown, under "Delete 'My Library'"

Changes made, back to you @iwehrman.

iwehrman commented on src/style/sections/libraries/libraries-section.less:
```diff
@@ -88,19 +88,27 @@
             border: 0;
-
+            width: 18.4rem;
+            border-bottom: @hairline solid fade(@color-subsection-rule, 60%);
```
Should we be `fade`ing colors here? Is that kosher? Is there a better way that will make our lives easier later when trying to maintain different color themes? CC @placegraphichere

placegraphichere replied:
lets not do this. I think @underline should work for this.

iwehrman commented:
Just one final question and then we're good to go.

iwehrman commented:
LGTM, thanks!
